.travis.yml

```diff
@@ -68,10 +72,25 @@ jobs:
     - script: make edge_integration_test
       name: "integration test edge"     # names the fourth Tests stage job
     - script: make cloud_test
-      name: "unit test cloud" 
+      name: "unit test cloud"
 
     - stage: "e2e tests"
-      script: make e2e_test
+      script:
+        - sudo apt-get update && sudo apt-get install -y apt-transport-https
+        - curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+        - echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list
+        - sudo apt-get update
+        - sudo apt-get install -y kubelet kubeadm kubectl
+        - sudo kubeadm init --pod-network-cidr=10.244.0.0/16
```

Comment:
We is the edgenode installation step?

Reply:
The edge node registration step is handled within the e2e cases

Reply:
That's not supposed behavior, we shall fix it then.